wp/sections/work.tex

```diff
@@ -0,0 +1,7 @@
+\subsection{ktlint}
+\par
+Describe how does work ktlint (4-5 sentences)
+
+\subsection{diKTat}
+\par
```

Comment:
When you will be talking about that say the following: 
diktat does AST-analysis. This means that for internal representation (IR) it uses Abstract Syntax Tree that was created from the parsed code by the kotlin-compiler.